### PR TITLE
Move metrics from Logging to Monitoring

### PR DESCRIPTION
**Pull request by pron**

### CODE REVIEW

**Comment by akullpp** [comment]
Please fix the merge conflict.

**Comment by pron** [comment]
Done.

File: README.md [hunk 1]
```diff
@@ -458,6 +457,7 @@ A curated list of awesome Java frameworks, libraries and software.
 * [JavaMelody](https://github.com/javamelody/javamelody) - Performance monitoring and profiling.
 * [jmxtrans](https://github.com/jmxtrans/jmxtrans/) - Tool to connect to multiple JVMs and to query them for their attributes via JMX. Its query language is based on JSON, which allows non-Java programmers to access the JVMs attributes. Likewise, this tool supports different output writes, including Graphite, Ganglia, StatsD, among others.
 * [Kamon](http://www.kamon.io/) - Tool for monitoring applications running on the JVM.
+* [Metrics](http://metrics.codahale.com/) - Expose metrics via JMX or HTTP and can send them to a database.
```
Comment (akullpp) [inline README.md]: See CI issues, the URL is wrong.

**Comment by pron** [comment]
Done.

**Pull request merged**